bindings/swift/src/lib.rs

```diff
@@ -208,16 +210,76 @@ pub unsafe extern "C" fn iota_listen(
     0
 }
 
-// #[no_mangle]
-// pub extern "C" fn iota_init_logger(file_name: *const c_char) {
-//     let c_file_name = unsafe {
-//         assert!(!file_name.is_null());
-//         CStr::from_ptr(file_name)
-//     };
-//     let _file_name = c_file_name.to_str().unwrap();
-
-//     let output_config = LoggerOutputConfigBuilder::new()
-//         .name(file_name)
-//         .level_filter(LevelFilter::Debug);
-//     init_logger(LoggerConfig::build().with_output(output_config));
-// }
+/// # Safety
+///
+/// `callback` will be invoked from another thread so context must be safe to `Send` across threads
+#[no_mangle]
+pub unsafe extern "C" fn clear_listeners(
```

Comment:
How does this name relate to the iota_ definitions ? Shouldn't this have an iota_prefix as well?

Reply:
Good catch! Thats not as intended and doesnt work, ive added the prefix

Reply:
Added an issue for swift bindings CI to catch that kind of stuff